sys/netbsd/lwp.txt

```diff
@@ -28,6 +29,19 @@ _lwp_setname(target lwpid, name ptr[in, string])
 _lwp_getname(target lwpid, name ptr[out, string], size bytesize[name])
 _lwp_ctl(features int32, address ptr[in, ptr[in, lwpctl]])
 
+resource semid[intptr]
+
+_ksem_init(val int32, idp ptr[out, semid])
```

Comment:
This is not fully `out` parameter.

It first accepts either 0 or `KSEM_PSHARED` for pshared. Later it returns `semid`.

Reply:
https://nxr.netbsd.org/xref/src/lib/librt/sem.c#197